woocommerce/woocommerce-android
# Feature/tracking profiler steps

Sorry for the short notice on this one 🙏🏼 (should be a quick one tough) 

### Description


The API to send the store creation profiler data is still WIP. But we want to start collecting metrics about the user when creating a store. This PR: 
- Enables profiler steps and country picker step in release
- Collects profiler data and selected country and send it as tracking data to Tracks

### Testing instructions
- Compile `vanillaDebug` flavour (this is needed otherwise IAP eligibility check will fail and you won't be able to proceed with the store creation flow).
- Trigger store creation and fill all the information requested in each step. Remember what your selections are when going through the profiler steps so you can verify that the values sent to Tracks are correct later on. 
- On the Plan screen click on "Create Store for 69.99/month"
- Check in the logs that the following event with the previously selected values is tracked: 
``` 🔵 Tracked: site_creation_profiler_data, Properties: {"industry_group":"automotive","industry":"car_washes","user_commerce_journey":"im_already_selling_online","ecommerce_platforms":"big-commerce","country_code":"DZ","is_debug":true}```

Note that after clicking on the button "Create Store for 69.99/month", the store creation will fail as you won't have the IAP testing environment setup. But the site will be created anyways and thus the tracking should be fired. 

### Images/gif
